lib/booking_manager.rb

```diff
@@ -0,0 +1,59 @@
+require_relative 'reservation.rb'
+require 'pry'
+
+class BookingManager
+
+  attr_reader :rooms
+  attr_accessor :reservations, :start_date, :end_date, :make_reservation
```

Comment:
We use `attr_`... methods at the top of classes in order to connect _instance variables_ to levels of access-- reading, writing, or both.

`BookingManager` doesn't have an instance variable named `@make_reservation`. It has a method named `make_reservation`, but methods don't belong here.

What were you trying to do?

Let me or Dan know if you need to sit down with someone to work through this misunderstanding.